cirq-google/cirq_google/engine/engine_job_test.py

```diff
@@ -505,7 +505,8 @@ def test_delete(delete_job):
 @mock.patch('cirq_google.engine.engine_client.EngineClient.get_job_results')
 def test_results(get_job_results):
     qjob = quantum.QuantumJob(
-        execution_status=quantum.ExecutionStatus(state=quantum.ExecutionStatus.State.SUCCESS)
+        execution_status=quantum.ExecutionStatus(state=quantum.ExecutionStatus.State.SUCCESS),
+        update_time=datetime.datetime.now(),
```

Comment:
Should this have a timezone since we've decided to enforce that for serialization? We should avoid naive datetimes in tests if we're trying to discourage them in other parts of the library.
```suggestion
        update_time=datetime.datetime.now(tz=datetime.timezone.utc),
```